src/manager.js

```diff
@@ -701,19 +715,37 @@ AdManager.prototype.refreshSlots = function (slotsToLoad) {
     return;
   }
 
+  var useIAS = typeof this.__iasPET !== 'undefined' && this.options.iasEnabled;
   var useIndex = typeof window.headertag !== 'undefined' && window.headertag.apiReady === true;
   var usePrebid = typeof window.pbjs !== 'undefined' && this.options.prebidEnabled;
+  var refreshMethod = function(){
+    if (useIndex) {
+      return window.headertag.pubads().refresh(slotsToLoad, {changeCorrelator: false});
+    } else if (usePrebid) {
+      return this.prebidRefresh(slotsToLoad);
+    } else {
+      return this.googletag.pubads().refresh(slotsToLoad, {changeCorrelator: false});
+    }
+  }
+
 
-  if (useIndex) {
-    window.headertag.pubads().refresh(slotsToLoad, {
-      changeCorrelator: false
+  if (useIAS){
+    var iasSlotSetup = require('./ias-slot-setup');
+
+    var gtSlots = this.googletag.pubads().getSlots();
+    var iasPETSlots = iasSlotSetup(gtSlots);
+    var iasRequestTimeout = setTimeout(refreshMethod(), this.options.iasTimeout);
```

Comment:
I don't think this is valid, `setTimeout` expects you to hand it a function declaration, but you're not supposed to immediately invoke it. 

I think this should be simply `refreshMethod` instead of `refreshMethod()`